### PR TITLE
Added final components to listing description.

### PR DESCRIPTION
**Pull request by tonaramos**
Test file added, but actual testing pending.
Database was updated to fix minor spelling errors. 
Code clean up.

### CODE REVIEW

File: client/src/js/components/AllAmenities.jsx [hunk 1]
```diff
@@ -1,60 +1,47 @@
...
-        <div className="title_10" >Amenities</div>
-        <div className="amenityTable" >
-          {allAmenitiesList}
+      <div key={category.title}>
+        <div className="amenityCategorySection">
+          <div className="title_11" >{category.title}</div>
         </div>
-      </hgroup>
+        <div className="itemsSection">
+          <AmenitySectionListing data={category.contents} />
```
Comment (rtsuzaki) [inline client/src/js/components/AllAmenities.jsx:31]: Consider using a different name than data as that name could get confused with the normal data attribute

File: client/src/js/components/App.jsx [hunk 2]
```diff
@@ -21,7 +23,7 @@ class App extends React.Component {
   }
 
   getHomeData(id) {
-    axios.get(`http://127.0.0.1:3001/homes/${id}`)
+    axios.get(`http://127.0.0.1:3001/rooms/${id}`)
```
Comment (rtsuzaki) [inline client/src/js/components/App.jsx:26]: use relative path for deployment

File: client/src/js/components/App.jsx [hunk 3]
```diff
@@ -51,17 +51,18 @@ class App extends React.Component {
...
             <HouseRules data={this.state.home} />
             <Cancellations data={this.state.home} />
+            <Availability data={this.state.home} />
           </div>
         </div>
       );
     }
     return (
-      <div>Loading...</div>
+      <div>{JSON.stringify(this.state.home)}</div>
```
Comment (rtsuzaki) [inline client/src/js/components/App.jsx:63]: ???

File: client/src/js/components/ViewsAlert.jsx [hunk 4]
```diff
@@ -3,12 +3,12 @@ import React from 'react';
 const ViewsAlert = (props) => {
   if (props.data > 200) {
     return (
-      <hgroup id="viewsAlert" className="lightBorder" >
-        <div className="viewsAlertText" >
-          <div className="title_1" >{'This home is on people\'s minds.'}</div>
-          <div id="ownerName" >{`It's been viewed ${props.data}+ times in the past week.`}</div>
+      <hgroup id="viewsAlert" className="lightBorder">
+        <div className="viewsAlertText">
+          <div className="title_4" >{'This home is on people\'s minds.'}</div>
```
Comment (rtsuzaki) [inline client/src/js/components/ViewsAlert.jsx:8]: Why do you need the text in brackets?

**Review by rtsuzaki — APPROVED** [review]
looks good!

File: client/src/js/components/ViewsAlert.jsx (hunk 4 above)
Comment (tonaramos) [reply client/src/js/components/ViewsAlert.jsx:8]: got the eslint, html entities must be scaped,  error due to the single quote in the text.

**Pull request merged**